input/pagecontent/change_log.md

```diff
@@ -1,3 +1,26 @@
+### FHIR Shorthand 3.0.0-ballot (HL7 Mixed Normative / Trial Use Ballot 1)
+
+The FHIR Shorthand 3.0.0 Mixed Normative / Trial Use Ballot (September 2023) introduced the following changes as **NORMATIVE** features. These features have been thoroughly tested by the community after being introduced as trial use in FHIR Shorthand 2.0.0 and are not expected to change in the future.
```

Comment:
> and are not expected to change in the future.

This could be confusing because we _are_ changing parameterized rulesets; we're just doing it in a backward-compatible way. Is there a better way to word this? A few possibilities:
* and are expected to remain stable in the future.
* and are not expected to undergo breaking changes in the future.

Reply:
Ah yes, that's a good idea, especially since we'll be documenting parameterized rulesets in the next section right below this. I had just used the same language as the previous entry but didn't think about this case. I think I like _"and are expected to remain stable in the future"_ best.